doc/source/tutorials/basictutorial.rst

```diff
@@ -0,0 +1,656 @@
+	Distance is farther than 1 kilometer.
+
+As you can imagine this isn't good. If we're at less than a meter away from something and the commands for if we would be farther
+than 1 kilometer show up we have a big problem. This could be fixed by doing the following, but **THIS IS UNNECESSARILY** complex: ::
+
+ set Done to false.
+
+ if Done = false {
+   if distance <= 1 {
+     print "Distance is within a meter.".
+     set NotDone to false.
+   }
+ }
+
+ if Done = false {
+   if distance <= 100 {
+     print "Distance is within 100 meters.".
+     set NotDone to false.
+   }
+ }
+
+ if Done = false {
+   if distance > 1000 {
+     print "Distance is farther than 1 kilometer.".
+     set NotDone to false.
+   }
+ }
+
+Now this essentially does the same as the ``else if`` script but it's way more confusing and complicating.
+
+====================
+Until, lock and wait
+====================
+
+The ``wait`` command pretty straight forward: ::
+
+  wait 10.
+ print "done waiting".
+
+
+It will take 10 seconds before ``done waiting`` shows up.
+Using ``wait 0`` will let the script wait for one physics tick, this can be handy for
+when you're doing stuff with maneuvers. Maneuvers don't show up instantly but show up
+after one physics tick. More about maneuvers in part ???.
+
+The ``until`` command will keep looping a piece of code until the given value has been met.
+Before we can talk about until loops let's first talk about ``time:seconds`` and the ``lock`` command. ::
+
+  print time:seconds.
+
+
+will print the current time in seconds. Let's say the time is 1 minute.
+It would print ``60``. You can also set the current time as a variable: ::
+
+  set CurrentTime to time:seconds.
+
+The variable ``CurrentTime`` will stay 60 seconds. Using the ``set`` command will look at
+a value and pick that value to stay the  same, even if the value it was set to changes.
+For instance, printing ``CurrentTime`` would give 60. Not only at ``time:seconds`` = 60,
+also at any other time like ``time:seconds`` = 4000. Eventhough ``time:seconds`` is 4000,
+``CurrentTime`` is still 60.
+
+The ``lock`` command updates constantly the variable, for example: ::
+
+  lock TimeSecondsPlusTen to time:seconds + 10.
+
+If you print ``TimeSecondsPlusTen`` at 60 seconds it will show 70, if you print
+``TimeSecondsPlusTen`` at 4000 seconds it will show 4010.
+
+Using until, lock and wait in an example
+_________________________________________
+
+If we now combine all the command we can make the following piece of code: ::
+
+  set Adder to 0.
+  lock Multiplier to Adder * 2.
+  set TimePlusFive to time:seconds + 5.
+
+  until time:seconds > TimePlusFive {
+    print Multiplier.
+    set Adder to Adder + 1.
+    wait 1.
+  }
+
+So an easy way to read the until loop is to cover what ever is inside of the curly brackets
+and say: until ``time:seconds`` is bigger than our current time plus 5 seconds, print
+``Multiplier``, increase the value of ``Adder`` and wait 1 second.
```

Comment:
This use of "cover up what ever is inside of the curly brackets" doesn't make sense.  You tell the reader to cover up the body of the loop, then tell them they can look at it that way and say: printing ``Multiplier``. and increasing the value of ``Adder``, and "wait 1 second".  All of these are in the section you just got done explicitly telling the reader to cover up and ignore at the moment.  There needs to be a clean break where you say, "and now take your hand away and look at what is inside the curly brackets and .... (etc)" before you get to the stuff about Multiplier, Adder, and waiting 1 second.